<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsCommands</class>
 <widget class="QWidget" name="PrefsCommands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Commands</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_main">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_options">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="_output_window">
         <property name="title">
          <string>Output Window</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="DisplayMyInput">
            <property name="text">
             <string>Echo my &amp;input in</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorPickerButton" name="EchoColorsForeground">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="EchoColorsBackground_label">
            <property name="text">
             <string>on</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorPickerButton" name="EchoColorsBackground">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_speed_walking">
         <property name="title">
          <string>Speed Walking</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="_walk">
            <item>
             <widget class="QCheckBox" name="EnableSpeedWalk">
              <property name="text">
               <string>Enable &amp;speed walking with prefix:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SpeedWalkPrefix">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="_delay" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="SpeedWalkDelay_label">
               <property name="text">
                <string>Delay:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SpeedWalkDelay">
               <property name="suffix">
                <string> s</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="SpeedWalkFiller_label">
               <property name="text">
                <string>Filler:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="SpeedWalkFiller"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_command_stacking">
         <property name="title">
          <string>Command Stacking</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="EnableCommandStack">
            <property name="text">
             <string>Enable &amp;command stacking with prefix:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CommandStackCharacter">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_command_window">
         <property name="title">
          <string>Command Window</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFontComboBox" name="InputFont">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_font_options">
            <item>
             <widget class="QSpinBox" name="InputFontSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="suffix">
               <string> pt.</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QCheckBox" name="UseDefaultInputFont">
              <property name="text">
               <string>O&amp;verride with default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_color">
            <item>
             <widget class="QLabel" name="InputColorsForeground_label">
              <property name="text">
               <string>Colour:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorPickerButton" name="InputColorsForeground">
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InputColorsBackground_label">
              <property name="text">
               <string>on</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorPickerButton" name="InputColorsBackground">
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_spam_prevention">
         <property name="title">
          <string>Spam Prevention</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="EnableSpamPrevention">
            <property name="text">
             <string>Enable spam prevention</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="_spam_options" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="SpamMessage_label">
               <property name="text">
                <string>Send:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="SpamMessage"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="SpamLineCount_label">
               <property name="text">
                <string>Every</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="_spam_line_count">
               <item>
                <widget class="QSpinBox" name="SpamLineCount">
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="SuffixSpamLine_label">
                 <property name="text">
                  <string>identical commands</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_checkboxes">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="AutoRepeat">
         <property name="text">
          <string>Auto-repeat command</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LowerCaseTabCompletion">
         <property name="text">
          <string>Tab completion in lowercase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TranslateGerman">
         <property name="text">
          <string>Translate German characters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TranslateBackslashSequences">
         <property name="text">
          <string>Translate backslash sequences</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KeepCommandsOnSameLine">
         <property name="text">
          <string>Keep commands on prompt line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NoEchoOff">
         <property name="text">
          <string>Ignore &quot;Echo Off&quot; messages</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Command History</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="HistoryLines_label">
            <property name="text">
             <string>Keep</string>
            </property>
            <property name="buddy">
             <cstring>HistoryLines</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="HistoryLines">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SuffixHistoryLines_label">
            <property name="text">
             <string>lines</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="keyboard_preferences_button">
         <property name="text">
          <string>Keyboard Preferences...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tab_completion_button">
         <property name="text">
          <string>Tab Completion...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>UseDefaultInputFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>InputFont</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseDefaultInputFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>InputFontSize</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableSpeedWalk</sender>
   <signal>toggled(bool)</signal>
   <receiver>SpeedWalkPrefix</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableSpamPrevention</sender>
   <signal>toggled(bool)</signal>
   <receiver>_spam_options</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableCommandStack</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommandStackCharacter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnableSpeedWalk</sender>
   <signal>toggled(bool)</signal>
   <receiver>_delay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
